src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -0,0 +1,140 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import org.apache.mahout.cf.taste.common.TasteException;
+import org.apache.mahout.cf.taste.impl.model.file.FileDataModel;
+import org.apache.mahout.cf.taste.impl.neighborhood.ThresholdUserNeighborhood;
+import org.apache.mahout.cf.taste.impl.recommender.GenericUserBasedRecommender;
+import org.apache.mahout.cf.taste.impl.similarity.PearsonCorrelationSimilarity;
+import org.apache.mahout.cf.taste.model.DataModel;
+import org.apache.mahout.cf.taste.neighborhood.UserNeighborhood;
+import org.apache.mahout.cf.taste.recommender.RecommendedItem;
+import org.apache.mahout.cf.taste.recommender.Recommender;
+import org.apache.mahout.cf.taste.similarity.UserSimilarity;
+
+import java.io.*;
+import java.util.*;
+import java.util.zip.GZIPInputStream;
```

Comment:
After reading further about this, I found something interesting: http://stackoverflow.com/questions/5319016/for-java-can-i-import-all-packages-at-once . You are correct. That file is in the package `java.util.zip` and you are importing the classes only in `java.util`. If you could do that, then importing `java.*` would work too. I think the correct way is to handle the way you are doing, because you are using a lot of classes in util, but only one in zip.